src/panels/lovelace/cards/hui-glance-card.ts

```diff
@@ -88,17 +88,13 @@ export class HuiGlanceCard extends HassLocalizeLitMixin(LitElement)
       return html``;
     }
     const { title } = this.config;
-    const states = this.hass.states;
-    const entities = this.configEntities!.filter(
-      (conf) => conf.entity in states
-    );
 
     return html`
       ${this.renderStyle()}
       <ha-card .header="${title}">
         <div class="entities ${classMap({ "no-header": !title })}">
           ${repeat<EntityConfig>(
```

Comment:
We no longer need repeat. Repeat is really only useful if you need to re-order rows (for example when 1 entity used to go missing, all other entities would shift up). Since we now always show something for each configured entity, this can just become a `this.configEntities!.map(entityConf => This.renderEntity(entityConf)`